openapi-core/src/main/java/io/ballerina/openapi/core/generators/client/FunctionBodyGenerator.java

```diff
@@ -645,10 +646,12 @@ private void createRequestBodyStatements(boolean isHeader, List<StatementNode> s
 
         //Create Request statement
         Map.Entry<String, MediaType> next = iterator.next();
-        VariableDeclarationNode requestVariable = GeneratorUtils.getSimpleStatement("http:Request",
-                "request", "new");
-        statementsList.add(requestVariable);
-        if (next.getValue() != null) {
+        if (!next.getKey().contains(ANY_TYPE)) {
+            VariableDeclarationNode requestVariable = GeneratorUtils.getSimpleStatement("http:Request",
+                    "request", "new");
```

Comment:
Is it possible to reuse the defined constants for `http:Request` and `request` in here?

Reply:
Yes. Updated